hoomd/hpmc/external/user.py

```diff
@@ -1,7 +1,15 @@
 # Copyright (c) 2009-2022 The Regents of the University of Michigan.
 # Part of HOOMD-blue, released under the BSD 3-Clause License.
 
-"""User-defined external fields for HPMC simulations."""
+"""User-defined external fields for HPMC simulations.
```

Comment:
I can't remember, but isn't this only supported on CPU? If so we should document it here.

Reply:
This is documented on line 82:
```
    Note:
        `CPPExternalPotential` does not support execution on GPUs.
```